index.js

```diff
@@ -154,6 +154,16 @@ module.exports = function (host, port) {
       return send('add', null, null, file, cb)
     },
     cat: argCommand('cat'),
+    slurp: function (path, cb) {
+      this.cat(path, function (err, stream) {
+        if (err) return cb(err)
+        var buf = ''
+        stream
+          .on('error', cb)
+          .on('data', function (data) { buf += data })
+          .on('end', function () { cb(null, buf) })
+      })
+    },
```

Comment:
I'm not sure if it's really necessary to include a dependency when this is pretty straightforward code that doesn't have to deal with all the complexity that concat-stream handles.